### PR TITLE
Clarify permission to view and manage shortcuts

### PR DESCRIPTION
**Pull request by kariannemah**
cf https://github.com/dobtco/screendoor-v2/pull/3200

### CODE REVIEW

File: contents/articles/screendoor/evaluation/workflow_automation_shortcuts.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Press the Delete button to remove a shortcut.
 
 ### Reversing a shortcut
 
-Whenever you trigger a shortcut, a notification appears on the lower right-hand corner of the page. If you have the right permissions, click the Undo link inside the notification.
+Whenever you trigger a shortcut, a notification appears on the lower right-hand corner of the page. If you are a project admin or have permission to manage shortcuts, click the Undo link inside the notification.
```
Comment (ajb) [inline contents/articles/screendoor/evaluation/workflow_automation_shortcuts.md]: I think this is redundant. We can cut "are a project admin or"

**Comment by ajb** [comment]
So the currently version of the article is ambiguous ("the right permissions"), but given that "view shortcuts" can be assigned to users as a custom permission, and isn't limited to "Reviewer" users, the current article might even be more correct. What do you think? It seems to be like we could just leave it as-is.

**Comment by kariannemah** [comment]
Given recent support requests asking us to elaborate on what collaborators can or cannot do, I'd rather be more explicit than ambiguous about permissions.

I changed "if you're a project admin or have permission to review responses" to "if you have permission to rate responses or view shortcuts." This accounts for both project admins and collaborators with the custom permission.

Mention of project admins in the managing shortcuts copy have been :scissors:, too.

**Comment by ajb** [comment]
Sounds good!

**Pull request merged**